src/notFound.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>404 not found</title>
+</head>
+<body>
+
```

Comment:
The body of the HTML document is empty. Consider adding a message or content to inform users that the page they are looking for cannot be found, such as a heading or a paragraph with a 404 message.